### DISCUSSION TITLE
How to remove code block formatting?

### DISCUSSION THREAD

**Discussion by Abdillah** [outline/outline#2584] [Self Hosting] [answered]
When copying a text from code block, it gets formatted as code block (with triple ticks). How to undo this formatting?

**Comment by tommoor** [comment, marked as answer]
You can use the keyboard shortcut, `Ctrl + ⇧ + \` or you can paste the content into Outline as plain text (on macOS it's `cmd`+`shift`+`v` to do this.

**Reply by Abdillah** [reply]
Unfortunately, the Ctrl + ⇧ + \ not working for the reverse formatting, despite works for creating code block. I've checked on both Firefox and Chrome, as well as checking whether OS-level shortcut was interrupting.

**Reply by tommoor** [reply]
This will be fixed in next deploy.
